src/test/ui/traits/issue-95898.rs

```diff
@@ -0,0 +1,10 @@
+// Test for #95898: The trait suggestion had an extra `:` after the trait.
+// edition:2021
+
+#[rustfmt::skip]
```

Comment:
I don't think you need rustfmt::skip here. We configured rustfmt to not run on ui tests for this exact reason.

Reply:
You might need to turn off auto formatting from your IDE to remove this attribute.

Reply:
Removed `rustfmt::skip`, thanks :+1: